.eslintrc

```diff
@@ -1,19 +1,25 @@
 {
-    "parserOptions": {
-        "ecmaVersion": 2019,
-        "sourceType": "module"
-    },
-    "env": {
-        "es6": true,
-        "browser": true
-    },
-    "plugins": [
-        "svelte3"
-    ],
-    "overrides": [
-        {
-            "files": ["*.svelte"],
-            "processor": "svelte3/svelte3"
-        }
-    ]
-}
\ No newline at end of file
+  "parserOptions": {
+    "ecmaVersion": 2019,
+    "sourceType": "module"
+  },
+  "env": {
+    "es6": true,
+    "browser": true
+  },
+  "plugins": ["svelte3", "prettier"],
+  "overrides": [
+    {
+      "files": ["*.svelte"],
+      "processor": "svelte3/svelte3",
+      "rules": {
+        "import/first": "off",
+        "import/no-duplicates": "off",
+        "import/no-mutable-exports": "off",
+        "import/no-unresolved": "off",
+        "import/prefer-default-export": "off"
```

Comment:
What do we turn off the following rules?

- `import/first`
- `import/no-duplicates`
- `import/no-mutable-exports`
- `import/no-unresolved`

Reply:
So that the eslint svelte plugin works properly: https://github.com/sveltejs/eslint-plugin-svelte3/blob/master/OTHER_PLUGINS.md

Reply:
Can we switch `.eslintrc` to `.eslintrc.js` and add your explanation as comment? It would be very helpful in the future in case we revisit this file.